doc/nrf/includes/sample_board_rows.txt

```diff
@@ -104,7 +104,7 @@
 
 .. nrf21540dk_nrf52840
 
-| nRF21540 DK | PCA10112 | :ref:`nrf21540dk_nrf52840 <nrf21540dk_nrf52840>` | ``nrf21540dk_nrf52840`` |
+| :ref:`nRF21540 DK <ug_radio_fem>` | PCA10112 | :ref:`nrf21540dk_nrf52840 <nrf21540dk_nrf52840>` | ``nrf21540dk_nrf52840`` |
```

Comment:
Seems kind of strange to link to the FEM page here. I can see that that page describes the nrf21540 DK. Would it make sense to have a separate "working with nrf21540" page and have the FEM page link to it instead?
AFAIK FEM does not have to be nrf21540.

Reply:
Consulted @greg-fer about this and he says: "This was not added because Eagle was in the pre-launch limbo. I'd say we could use the link to ug_radio_fem, to stay consistent with other links in the table, which also link to ug_ for their respective DKs." The FEM page links you further to other documents about nRF21540 DK.

Reply:
ug_radio_fem is not the ug_ for the nrf21540 DK. It is the user guide for the entire FEM feature. There are many front-end modules, and nrf21540 is one of them.
It would make more sense to me to document the nrf21540 DK separately. And link to it from the FEM user guide.

Reply:
I agree, it is not the UG for the DK. However, this is a bit outside the scope of this PR. How about then linking to the documentation we have in the infocenter, such as https://infocenter.nordicsemi.com/index.jsp?topic=%2Fstruct_fem%2Fstruct%2Fnrf21540.html?

Reply:
We don't link to Infocenter for other DKs that have "Working with" pages in the NCS, so I don't think we should do this for nRF21540, also because "Working with" pages contain information about using the given hardware in the NCS context, whereas Infocenter pages do not.

For me, we should either link to ug_radio_fem (which is the closest to a Working with nRF21540 we have right now) or take a step back (close this PR), rework ug_radio_fem into a Working with guide, and then add the link when we have the guide.

Reply:
So, @peknis , maybe we link to ug_radio_fem?

Reply:
Fine for me, at least for now as we do not have a proper working with FEM guide. What do you think @joerchan?

Reply:
> For me, we should either link to ug_radio_fem (which is the closest to a Working with nRF21540 we have right now) or take a step back (close this PR), rework ug_radio_fem into a Working with guide, and then add the link when we have the guide.

I think you could take a step forward, and merge the PR, and then improve it later.

I don't have a strong opinion though, so I'm happy with whatever you guys think is best.